certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          }
+        },
+        "titleCompany": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "region": {
+          "description": "Gets or Sets Region",
+          "type": "string"
+        },
+        "branch": {
+          "description": "Gets or Sets Branch",
+          "type": "string"
+        },
+        "agents": {
+          "description": "Gets or Sets Agents",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+          }
+        },
+        "underwriter": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "escrowBank": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "sourceOfBusiness": {
+          "description": "Gets or Sets SourceOfBusiness",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.ProductDetail": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "dates": {
+          "description": "Gets or Sets Dates",
+          "type": "array",
+          "items": {
+            "format": "date-time",
+            "type": "string"
+          }
+        },
+        "productReferenceKey": {
+          "format": "uuid",
+          "description": "Gets or Sets ProductReferenceKey",
+          "type": "string"
+        },
+        "submissionReference": {
+          "description": "Gets or Sets SubmissionReference",
+          "type": "string"
+        },
+        "code": {
+          "description": "Gets or Sets Code",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or Sets Description",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderAddressType": {    
+      "description": "Gets or Sets TitleOrderAddressType",
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.County": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or Sets Name",
+          "type": "string"
+        },
+        "fips": {
+          "description": "Gets or Sets Fips",
+          "type": "string"
+        },
+        "statecode": {
+          "format": "int64",
+          "description": "Gets or Sets Statecode",
+          "type": "integer"
+        },
+        "countycode": {
+          "format": "int64",
+          "description": "Gets or Sets Countycode",
+          "type": "integer"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderAddress": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.